caddyhttp/fastcgi/setup.go

```diff
@@ -2,10 +2,14 @@ package fastcgi
 
 import (
 	"errors"
+	"fmt"
 	"net/http"
 	"path/filepath"
+	"strings"
 	"time"
 
```

Comment:
delete whitespace and gofmt to rearrange imports.

Reply:
@abiosoft both fixed